OpenRA.Mods.Common/Widgets/Logic/ReplayBrowserLogic.cs

```diff
@@ -604,15 +596,19 @@ void SelectFirstVisibleReplay()
 		void SelectReplay(ReplayMetadata replay)
 		{
 			selectedReplay = replay;
-			selectedSpawns = (selectedReplay != null)
-				? LobbyUtils.GetSpawnOccupants(selectedReplay.GameInfo.Players, selectedReplay.GameInfo.MapPreview)
-				: new Dictionary<CPos, SpawnOccupant>();
+			Map = (selectedReplay != null) ? selectedReplay.GameInfo.MapPreview : MapCache.UnknownMap;
 
 			if (replay == null)
 				return;
 
 			try
 			{
+				if (Map.Status != MapStatus.Available)
```

Comment:
Style nit: don't leave hanging blocks.  The hanging else-if is ambiguous between the inner if and the outer if (with an indentation error), so this needs braces to make it explicit.